src/shared/conf-parser.c

```diff
@@ -716,10 +716,8 @@ int config_parse_path(
         assert(rvalue);
         assert(data);
 
-        if (isempty(rvalue)) {
```

Comment:
I have an **identical** commit in #9261. But even if both go in, the merge should not be a problem.